From 126be8a3867aa70459ad37918a306679484d406b Mon Sep 17 00:00:00 2001
From: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Date: Tue, 21 Sep 2021 17:18:31 +0200
Subject: [PATCH 038/529] clk: stm32mp1: use clk_hw_set_parent instead
 clk_hw_reparent()

When doing a multi-mux operation, use clk_hw_set_parent(),
this way the change will be seen on the clock summary.

Signed-off-by: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
---
 drivers/clk/clk-stm32mp1.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/clk/clk-stm32mp1.c
+++ b/drivers/clk/clk-stm32mp1.c
@@ -728,7 +728,7 @@ static int clk_mmux_set_parent(struct cl
 
 	for (n = 0; n < clk_mmux->mmux->nbr_clk; n++)
 		if (clk_mmux->mmux->hws[n] != hw)
-			clk_hw_reparent(clk_mmux->mmux->hws[n], hwp);
+			clk_hw_set_parent(clk_mmux->mmux->hws[n], hwp);
 
 	return 0;
 }
